internal/flags/categories.go

```diff
@@ -37,6 +37,7 @@ const (
 	MiscCategory       = "MISC"
 	TestingCategory    = "TESTING"
 	DeprecatedCategory = "ALIASED (deprecated)"
+	GuardianCategory   = "Guardian"
```

Comment:
nit: ALL uppercase to follow the same convention

Reply:
Done